R/mcmc-intervals.R

```diff
@@ -291,6 +293,7 @@ mcmc_areas <- function(x,
                        prob_outer = 1,
                        point_est = c("median", "mean", "none"),
                        rhat = numeric(),
+                       size = NULL,
```

Comment:
should we have a more descriptive name than size? "border_size"?

Reply:
Good call. Changing to `border_size`.